Fixes #115: Broken Internal Links

GwendolynK opened the pull request:
Found a few broken internal links on old news entries. Some of the pages, like the FAQ, did not have a clear successor/new location, so put the closest option I could find.

The rest of the changes were either fixing the links (adding `https://`) or replacing with the best new link based on context.

domingoruiz commented:
@ressonix How about it?

hbjydev commented:
@domingoruiz We're reaching the point where manual translation is _way_ too difficult to maintain. There are far too many languages already supported, and trying to get people to translate our static content would just take **far** too long.

I think I'm going to close this PR (for one, it has no attached issue, so it doesn't meet our contribution guidelines, and for two, we need to think of a permanent solution before we write a couple of scripts to hack together a solution).

However, feel free to open an issue for this, as this is something we do eventually need to fix.

domingoruiz commented:
I don't think he understood me. My intention is not to translate all the posts because that would be crazy. my only intention is that the site works well. Right now the posts do not work in any language, it only works in English, returning a 404 in the rest of the languages ​​and with this solution, which is the only one that hugo allows, it works the posts in all languages.

hbjydev commented:
@domingoruiz Can you give me a URL example?

But still open an issue please, we need to meet the contribution guidelines.

domingoruiz commented:
#146 read it

hbjydev commented:
Alright. After a quick discussion with @ressonix, we've decided that we do _want_ a solution to this problem, but we're not sure a quick hacky fix is that solution.

We might revisit this later, but for now we're going to leave this. I'm keeping the issue on the tracker though.

domingoruiz commented:
The problem is unsolved and must be solved. I've been reading Hugo's documentation looking for alternatives but this is the only one I found. If you find a better one, if none is found, we reopen this pull request, because as you have been able to check if it works. thanks.

hbjydev commented:
Yes. It must be solved. That does not mean we will use this solution whether we find a better one or not.

GwendolynK committed:
Fixes #115: Broken Internal Links

Found a few broken internal links on old news entries. Some of the
pages, like the FAQ, did not have a clear successor/new location, so put
the closest option I could find.

The rest of the changes were either fixing the links (adding `https://`)
or replacing with the best new link based on context.
